pytorch_lightning/utilities/xla_device_utils.py

```diff
@@ -41,11 +40,12 @@ def wrapper(*args, **kwargs):
         queue = Queue()
         proc = Process(target=inner_f, args=(queue, func, *args), kwargs=kwargs)
         proc.start()
-        proc.join(10)
+        proc.join(20)
         try:
             return queue.get_nowait()
         except q.Empty:
-            return False
+            traceback.print_exc()
+            return None
```

Comment:
not sure, would exist work be better?

Reply:
as in? I changed this to return `None` just to know when the tpu check timed out.